lib/features/splash/presentation/pages/splash_screen.dart

```diff
@@ -42,7 +42,8 @@ class _SplashScreenState extends State<SplashScreen>
           if (snapshot == const SplashState.none()) {
             context.router.popAndPush(const LoginScreenRoute());
           } else {
-            context.router.popAndPush(const HomeScreenRoute());
+            // context.router.popAndPush(const HomeScreenRoute());
```

Comment:
remove the comment

Reply:
so the home screen should have a button to open the judge screen

Reply:
Done!